<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MsgNewWindow</class>
 <widget class="QWidget" name="MsgNewWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>49</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>标题：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="content">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>82</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>内容：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancle">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_remarks">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>166</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>备注：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_remarks">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>166</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_modify">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
